src/openforms/registrations/contrib/objects_api/plugin.py

```diff
@@ -161,7 +160,29 @@ def register_submission(
                 get_drc=get_drc,
                 language=language_code_2b,
             )
-            record_data["csv_url"] = submission_csv_document["url"]
+            csv_url = submission_csv_document["url"]
+
+        base_render_context = {
+            "_submission": submission,
+            **get_variables_for_context(submission),
+            "type": options["productaanvraag_type"],
+            "submission": {
+                "uuid": submission.uuid,
+                "language_code": submission.language_code,
+                "attachments": attachments,
+                "pdf_url": document["url"],
+                "csv_url": csv_url,
+            },
```

Comment:
why this nesting? I think it becomes a bit confusing to have both `submission` and `_submission`. I am not sure if we need the uuid at all, but I think that the attachments, language code and the urls can appear in the top level?

Reply:
I asked for the nesting, as namespacing, to prevent conflicts but I'll leave this up to you and Sergei to decide.